HyperV/scripts/create-environment.ps1

```diff
@@ -202,39 +209,55 @@ function cherry_pick($commit) {
     $ErrorActionPreference = $eapSet
 }
 
+Write-Host "BuildDir is: $buildDir"
+Write-Host "ProjectName is: $projectName"
+Write-Host "Listing $buildDir parent directory:"
+Get-ChildItem ( Get-Item $buildDir ).Parent.FullName
+Write-Host "Listing $buildDir before install"
+Get-ChildItem $buildDir
+
 ExecRetry {
-    pushd C:\OpenStack\build\openstack\neutron
-    & pip install C:\OpenStack\build\openstack\neutron
+    Write-Host "Content of $buildDir\neutron"
+    pushd $buildDir\neutron
+    & pip install $buildDir\openstack\neutron
     if ($LastExitCode) { Throw "Failed to install neutron from repo" }
     popd
 }
 
 ExecRetry {
-    pushd C:\OpenStack\build\openstack\networking-hyperv
-    & pip install C:\OpenStack\build\openstack\networking-hyperv
+    Write-Host "Content of $buildDir\networking-hyperv:"
+    Get-ChildItem $buildDir\networking-hyperv
+    pushd $buildDir\networking-hyperv
+    & pip install $buildDir\networking-hyperv
     if ($LastExitCode) { Throw "Failed to install networking-hyperv from repo" }
     popd
 }
 
 ExecRetry {
-    pushd C:\OpenStack\build\openstack\nova
-    & pip install C:\OpenStack\build\openstack\nova
+    Write-Host "Content of $buildDir\nova"
+    Get-ChildItem $buildDir\nova
+    pushd $buildDir\nova
+    & pip install $buildDir\nova
     if ($LastExitCode) { Throw "Failed to install nova fom repo" }
     popd
 }
 
 ExecRetry {
-    pushd C:\OpenStack\build\openstack\compute-hyperv
-    & pip install C:\OpenStack\build\openstack\compute-hyperv
+    Write-Host "Content of $buildDir\compute-hyperv"
+    Get-ChildItem $buildDir\compute-hyperv
+    pushd $buildDir\compute-hyperv
+    & pip install $buildDir\compute-hyperv    
     if ($LastExitCode) { Throw "Failed to install Hyperv-Compute fom repo" }
     popd
 }
 
 ExecRetry {

```

Comment:
os-win setup should be before the others (as it's a library required by the other projects).